### ISSUE TITLE
UC Backups: deployed CRDs are older than deployed velero version

### ISSUE THREAD

**Issue by judge-red** [kubermatic/kubermatic#13877]
### What happened?

I noticed this weird error:
```
$ velero backup describe test-1 | tail -n2
Backup Volumes:
  <error getting backup volume info: DownloadRequest.velero.io "test-1-2ea6412a-e1f2-4312-8306-5d3bfd5267f4" is invalid: spec.target.kind: Unsupported value: "BackupVolumeInfos": supported values: "BackupLog", "BackupContents", "BackupVolumeSnapshots", "BackupItemOperations", "BackupResourceList", "BackupResults", "RestoreLog", "RestoreResults", "RestoreResourceList", "RestoreItemOperations", "CSIBackupVolumeSnapshots", "CSIBackupVolumeSnapshotContents">
```

And it's true, `BackupVolumeInfos` isn't supported by the `DownloadRequest.velero.io` CRD:
```
$ k get crd downloadrequests.velero.io -o yaml | grep -i BackupVolumeInfos | wc -l
       0
```

It appears Velero 1.14.0 was deployed:
```
$ k -n velero get deploy velero -o yaml | grep image:
        image: docker.io/velero/velero:v1.14.0
      - image: docker.io/velero/velero-plugin-for-aws:v1.10.0
```

However, the v1.14.0 CRD does indeed support it:
```
$ curl -s https://raw.githubusercontent.com/vmware-tanzu/velero/refs/heads/release-1.14/config/crd/v1/bases/velero.io_downloadrequests.yaml | grep -i BackupVolumeInfos | wc -l 
       1
```

### Expected behavior

CRD versions should match deployment version

### How to reproduce the issue?

Not sure, but presumably:
- enable and configure UC backups for a UC
- create a backup and enable volumes
- `velero backup describe <backup name>`

### How is your environment configured?

- KKP version: 2.26.0
- Shared or separate master/seed clusters?: shared

### Provide your KKP manifest here (if applicable)

N/A

### What cloud provider are you running on?

N/A

### What operating system are you running in your user cluster?

N/A

**Comment by judge-red** [comment]
@xrstf This was indirectly fixed by https://github.com/kubermatic/kubermatic/pull/13897, right?

**Comment by xrstf** [comment]
Actually it was fixed right after via #13898. Both were backported to 2.26 via #13901.

**Comment by judge-red** [comment]
Alright, then let's close this issue. Thanks!
